ops/configurator/Dockerfile

```diff
@@ -1,8 +1,29 @@
-FROM frolvlad/alpine-oraclejdk8
-MAINTAINER Nikita Volkov
-COPY fabric-configurator-1.2.16.jar fabric-configurator-1.2.16.jar
+#steps to get OpenJDK8
+FROM alpine:3.9
```

Comment:
ca we use "openjdk:8-alpine" instead of setting up java manually?

Reply:
Fixed